package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "universal-authenticator-library",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
This should probably be a minor bump since this is a new non-optional function to implement

Reply:
👍  updated